models/repo.go

```diff
@@ -593,15 +593,19 @@ func (repo *Repository) getReviewers(e Engine, doerID, posterID int64) ([]*User,
 	}
 
 	// This is a "public" repository:
-	// Any user that has write access or who is a watcher can be requested to review
+	// Any user that has write access, is a watcher or organization member can be requested to review
```

Comment:
why extra comma?

Reply:
If the comma is dropped the sentence is ambiguous "any user that has write access is a watcher".
I can change it back to "or who" if that's preferred.